tests/app/notifications/test_process_notification.py

```diff
@@ -327,7 +327,7 @@ def test_simulated_recipient(notify_api, to_address, notification_type, expected
 @pytest.mark.parametrize('recipient, expected_international, expected_prefix, expected_units', [
     ('7900900123', False, '44', 1),  # UK
     ('+447900900123', False, '44', 1),  # UK
-    ('07700900222', False, '44', 1),  # UK
+    ('07700900222', True, '44', 1),  # UK
```

Comment:
Should add a comment here to make it clear why `expectd_international` is `True` for a UK number:
```suggestion
    ('07700900222', True, '44', 1),  # UK (Jersey)
```